tests/integration/targets/setup_anycast_config/tasks/main.yml

```
@@ -0,0 +1,27 @@
+---
+- module_defaults:
+    group/infoblox.universal_ddi.all:
+      portal_url: "{{ portal_url }}"
+      portal_key: "{{ portal_key }}"
+  block:
+    # Create a random Auth Zone to avoid conflicts
+    - ansible.builtin.set_fact:
+        _anycast_config_name: "anycast_config_name-{{ 999999 | random | string }}"
```

```suggestion
        anycast_config_name: "anycast_config_name-{{ 999999 | random | string }}"
```
Just for code consistency with other setup files

---

done